src/views/Nft/components/TransferNftModal.tsx

```diff
@@ -51,21 +51,18 @@ const TransferNftModal: React.FC<TransferNftModalProps> = ({ nft, tokenIds, onSu
       if (!isValidAddress) {
         setError(TranslateString(999, 'Please enter a valid wallet address'))
       } else {
-        await pancakeRabbitsContract.methods
-          .transferFrom(account, value, tokenIds[0])
-          .send({ from: account })
-          .on('sending', () => {
-            setIsLoading(true)
-          })
-          .on('receipt', () => {
-            onDismiss()
-            onSuccess()
-          })
-          .on('error', () => {
-            console.error(error)
-            setError('Unable to transfer NFT')
-            setIsLoading(false)
-          })
+        setIsLoading(true)
+        const tx = pancakeRabbitsContract
+          .transferFrom(account, value, tokenIds[0], { from: account })
+        try {
+          tx.wait()
```

Comment:
Is there a reason to use this over async?

Reply:
Line 58 seems wrong. It should be like `await tx.wait()`